nemo_skills/inference/prompt/few_shot_examples/examples_tabmwp.py

```diff
@@ -0,0 +1,217 @@
+Hector | 80
+Devin | 83
+Emily | 82
+Avery | 87
+""".strip(),
+        "reference_solution": "Read the numbers from the table.\n\n76, 94, 86, 84, 80, 83, 82, 87\n\nFirst, count how many numbers are in the group.\n\nThere are 8 numbers.\n\nNow add all the numbers together:\n\n76 + 94 + 86 + 84 + 80 + 83 + 82 + 87 = 672\n\nNow divide the sum by the number of numbers:\n\n672 \u00f7 8 = 84\n\nThe mean is 84.",
+        "generated_solution": """
+Let's solve this problem using Python code and the values from the table.
+{start_code}
+number_of_coins_for_different_person = [76, 94, 86, 84, 80, 83, 82, 87]
+result = sum(number_of_coins_for_different_person) / len(number_of_coins_for_different_person)
+result
+{end_code}
+{start_code_output}
+84
+{end_code_output}
+So the mean of the sizes of their coin collections is \\boxed{84} coins.
+""".strip(),
+    },
+    {
+        "question": """Read the table regarding "" and then write Python code to answer the following question:
+
+Question: Look at the table. Then answer the question. At a price of $155, is there a shortage or a surplus? Choose from the the options: [shortage, surplus]""",
+        "table": """
+Price | Quantity demanded | Quantity supplied
+$155 | 22,600 | 5,800
+$275 | 20,500 | 9,400
+$395 | 18,400 | 13,000
+$515 | 16,300 | 16,600
+$635 | 14,200 | 20,200
+""".strip(),
+        "reference_solution": "At the price of $155, the quantity demanded is greater than the quantity supplied. There is not enough of the good or service for sale at that price. So, there is a shortage.\n",
+        "generated_solution": """
+Let's solve this problem using Python code and the values from the table.
+{start_code}
+quantity_demanded_price_155 = 22600
+quantity_supplied_price_155 = 5800
+if quantity_demanded_price_155 > quantity_supplied_price_155:
+    result = 'shortage'
+else:
+    result = 'surplus'
+result
+{end_code}
+{start_code_output}
+surplus
+{end_code_output}
+So there is a \\boxed{surplus} at a price of 155 dollars.
+""".strip(),
+    },
+    {
+        "question": """Read the table regarding "Cans of food collected" and then write Python code to answer the following question:
+
+Question: Samir's class recorded how many cans of food each student collected for their canned food drive. What is the median of the numbers?""",
+        "table": """
+Samir | 7
+Kristen | 4
+Dakota | 7
+Jamie | 8
+Maggie | 9
+""".strip(),
+        "reference_solution": "Read the numbers from the table.\n\n7, 4, 7, 8, 9\n\nFirst, arrange the numbers from least to greatest:\n\n4, 7, 7, 8, 9\n\nNow find the number in the middle.\n\n4, 7, 7, 8, 9\n\nThe number in the middle is 7.\n\nThe median is 7.",
+        "generated_solution": """
+Let's solve this problem using Python code and the values from the table.
+{start_code}
+cans = [7, 4, 5, 8, 9]
+cans = sorted(cans)
+middle1 = (len(cans) - 1) // 2
+middle2 = len(cans) // 2
+result = (cans[middle1] + cans[middle2]) / 2
+result
+{end_code}
+{start_code_output}
+7
```

Comment:
I think the actual output that Python gives is "7.0". Suggest you double check it in all places to make sure the few-shot examples are more representative of what the model will see during real inference

Reply:
Oh, nice catch, thanks. Just ran all the example solutions to double-check the outputs and found a few more issues.